### PR TITLE
Prometheus metrics added to dashboard

### PR DESCRIPTION
**Pull request by wyb1**

### CODE REVIEW

**Comment by msohn** [comment]
I think you should squash the 3 commits of this PR into 1 commit

**Comment by petersutter** [comment]
I also scheduled a meeting with Frederik on monday..

File: backend/lib/prometheus/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const express = require('express')
+const router = module.exports = express.Router()
+const cors = require('cors')
+const { notFound, sendError } = require('../middleware')
+const bodyParser = require('body-parser')
+const client = require('prom-client')
+const collectDefaultMetrics = client.collectDefaultMetrics
+
+// collect metrics every 5 seconds
+collectDefaultMetrics({ timeout: 5000 })
```
Comment (mvladev) [inline backend/lib/prometheus/index.js]: Don't scrape every 5 seconds - by default we scrape metrics every 30 secs with our Prometheus instances

File: backend/lib/prometheus/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+'use strict'
+
+const express = require('express')
+const router = module.exports = express.Router()
```
Comment (petersutter) [inline backend/lib/prometheus/index.js]: module.exports = 
can be removed here as it's already exported at the bottom (line 40-42)

**Pull request merged**